api/tests/integration/infrastructure/repositories/participant-results-shared-repository_test.js

```diff
@@ -0,0 +1,121 @@
+const { expect, databaseBuilder, mockLearningContent } = require('../../../test-helper');
+
+const participantResultsSharedRepository = require('../../../../lib/infrastructure/repositories/participant-results-shared-repository');
+const Campaign = require('../../../../lib/domain/models/Campaign');
+
+describe('Integration | Repository | Campaign Participation Aggregate Repository', () => {
+
+  describe('#get', () => {
+    it('computes the participant results the complete referential', async () => {
+      const { id: campaignId } = databaseBuilder.factory.buildCampaign({ type: Campaign.types.PROFILES_COLLECTION });
+
+      const participation = _buildParticipationWithSnapshot(
+        { campaignId, sharedAt: new Date('2020-01-02') },
+        [
+          { skillId: 'skill_1', status: 'invalidated', earnedPix: 0 },
+          { skillId: 'skill_2', status: 'validated', earnedPix: 3 },
+          { skillId: 'skill_3', status: 'validated', earnedPix: 1 },
+        ],
+      );
+
+      await databaseBuilder.commit();
+
+      const learningContent = {
+        skills: [
+          { id: 'skill_1', status: 'actif' },
+          { id: 'skill_2', status: 'archivé' },
+          { id: 'skill_3', status: 'périmé' },
+        ],
+      };
+      mockLearningContent(learningContent);
+
+      //when
+      const participantResultsShared = await participantResultsSharedRepository.get(participation.id);
+
+      expect(participantResultsShared.id).to.equal(participation.id);
+      expect(participantResultsShared.masteryPercentage).to.equals(1 / 2);
+      expect(participantResultsShared.pixScore).to.equals(3);
+      expect(participantResultsShared.validatedSkillsCount).to.equals(1);
+    });
+
+    it('computes the participant results for a target profil', async () => {
```

Comment:
`computes the participant results for a target profile`

Reply:
done